compiler/rustc_const_eval/src/interpret/operand.rs

```diff
@@ -621,6 +621,10 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
             ty::ConstKind::Infer(..) | ty::ConstKind::Placeholder(..) => {
                 span_bug!(self.cur_span(), "const_to_op: Unexpected ConstKind {:?}", c)
             }
+
+            // FIXME(julianknodt): this is probably a panic? If it contains any params
+            ty::ConstKind::Abstract(_ac) => todo!(),
```

Comment:
It should probably either be TooGeneric or a span_bug -- not sure how this is all staged.